addons/glsmoke/CfgAmmo.hpp

```diff
@@ -0,0 +1,9 @@
+class CfgAmmo
+{
+	class SmokeShell;
+	class G_40mm_Smoke: SmokeShell
+	{
+		simulation = "shotSmoke";
+		deflecting = 0;
```

Comment:
Yeah, i tried a few, but it still behaves very weird then, sometimes it just slides on the ground and bounces uncontrollable at the end.